pyface/ui/qt4/workbench/split_tab_widget.py

```diff
@@ -324,7 +321,7 @@ def _focus_changed(self, old, new):
             else:
                 nw = ntw.widget(ntidx)
 
-            self.emit(QtCore.SIGNAL('hasFocus'), nw)
```

Comment:
Probably off-topic for this PR, but why `SIGNAL('hasFocus')` rather than `SIGNAL('hasFocus()')`? This seems inconsistent with the other uses of `SIGNAL`.